tests/regression/02-base/82-eq-no-overflow.c

```diff
@@ -0,0 +1,17 @@
+// PARAM: --enable ana.int.interval
+#include <assert.h>
+
+int main() {
+  unsigned long x;
+  x = 0;
+
+  int b;
+  b = x == 7; // NOWARN
```

Comment:
Is there a specific reason for this `NOWARN` annotation? Did we previously produce an overflow warning here?